functions.js

```diff
@@ -8,10 +8,12 @@ window.onload = (event) => {
   document.getElementById("warning3").style.display = 'none';
   document.getElementById("warning4").style.display = 'none';
   document.getElementById("label-success").style.display = 'none';
-
-// document.getElementById("blurMe").style.filter = "blur(6px)";
-
-  document.getElementById("opacityme").style.filter = "blur(6px)";
+  document.getElementById("blur-business-a").style.filter = "blur(6px)";
+  document.getElementById("blur-business-b").style.filter = "blur(6px)";
+  document.getElementById("blur-enterprise-a").style.filter = "blur(6px)";
+  document.getElementById("blur-enterprise-b").style.filter = "blur(6px)";
+  document.getElementById("blur-plus-a").style.filter = "blur(6px)";
+  document.getElementById("blur-plus-b").style.filter = "blur(6px)";
```

Comment:
Add conditional chaining to check if element with given id present in document
```suggestion
  document.getElementById("blur-business-a")?.style.filter = "blur(6px)";
  document.getElementById("blur-business-b")?.style.filter = "blur(6px)";
  document.getElementById("blur-enterprise-a")?.style.filter = "blur(6px)";
  document.getElementById("blur-enterprise-b")?.style.filter = "blur(6px)";
  document.getElementById("blur-plus-a")?.style.filter = "blur(6px)";
  document.getElementById("blur-plus-b")?.style.filter = "blur(6px)";
```